website/client/js/controllers/inviteToGroupCtrl.js

```diff
@@ -44,8 +44,13 @@ habitrpg.controller('InviteToGroupCtrl', ['$scope', '$rootScope', 'User', 'Group
 
       Groups.Group.invite($scope.group._id, invitationDetails)
         .then(function() {
-          Notification.text(window.env.t('invitationsSent'));
-          _resetInvitees();
+           if ((inviteMethod === 'email' && invitationDetails.emails.length > 1) || (inviteMethod === 'uuid' && invitationDetails.uuids.length > 1)) { // test if it's inviting more than 1 emails or uuid
+                Notification.text(window.env.t('invitationsSent'));
+            }
+            else {
+                Notification.text(window.env.t('invitationSent'));
+            }
+            _resetInvitees();
```

Comment:
Add tests to [`inviteToGroupCtrlSpec.js`](https://github.com/Ozzystrasza/habitrpg/blob/18d975469328fff7ac2798d32e70498e7c0eb57f/test/spec/controllers/inviteToGroupCtrlSpec.js) that test these changes.